deps.ts

```diff
@@ -1,4 +1,4 @@
-export * from "https://raw.githubusercontent.com/lucacasonato/fresh/04a32f34987fbd59ce646967f216b2cca072a58f/runtime.ts";
+export * from "https://raw.githubusercontent.com/piscisaureus/fresh/d990a0bd6b76b2c2ec4016e9bf2584c7c5c8fe88/runtime.ts";
```

Comment:
This is getting a little ridiculous - we need to have a normal review & release process for code going into fresh if we're going to depend on it for our core infrastructure. 

cc @lucacasonato @piscisaureus

Reply:
we moved to bert's fork since luca isnt around to review & merge PRs

Reply:
Use https://raw.githubusercontent.com/denoland/fresh/0.0.2/runtime.ts

This doesn't include bert's etag patch - we should fix https://github.com/denoland/deploy/issues/2758 before landing this one.